bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -44,7 +44,7 @@
   }
 
   public static Configuration configure(String projectId, String zoneName, String clusterName) {
-    Configuration config = new Configuration();
```

Comment:
A user observed that there are lots of hanging threads in https://github.com/GoogleCloudPlatform/cloud-bigtable-client/issues/630#issuecomment-174023573.  It indicates that new Configuration() tries to find XML files that may be in jars.  That's expensive in terms of time and locking behavior.

Our intended default behavior in dataflow is to have programmatic configuration of the connection and to avoid lookups in xml files, since XML files are harder to set up for this situation.

See https://hadoop.apache.org/docs/r2.6.1/api/org/apache/hadoop/conf/Configuration.html#Configuration(boolean) for more details.